src/test/unit/gov/nist/javax/sip/stack/reInvite/Shootist.java

```diff
@@ -97,6 +97,8 @@
 
 	public boolean isTargetRefresh;
 
+	public boolean isAckWithSameBranch;
```

Comment:
i assume we expect this to be "false" by default,so it only applies to new test...